### ISSUE TITLE
The `length_is` template filter will be removed in 5.1 of Django.

### ISSUE THREAD

**Issue by jamesgilmorelyst** [farridav/django-jazzmin#535]
To add support for Django 5.1, the `length_is` feature is deprecated in 4.2 and is being removed in 5.1 [0], so it needs to be fixed.

Warning message when running Django-Jazzmin in v4.2 of Django:
```
RemovedInDjango51Warning: The length_is template filter is deprecated in favour of the length template filter and the == operator within an {% if %} tag.
```

[0] https://docs.djangoproject.com/en/dev/releases/5.1/#:~:text=The%20length_is%20template%20filter%20is%20removed.

**Comment by jamesgilmorelyst** [comment]
Raised a PR to fix this issue:
https://github.com/farridav/django-jazzmin/pull/534

@farridav could you take a look at this PR, if you have time, please?

**Comment by pyaidev** [comment]
Uninstall your current version of Django
Remove the current version of Django from your project:
`pip uninstall django`
install a specific version of Django
Install the desired version of Django (for example, 5.0):
`pip install django==5.0`
I try and fix the problem

**Comment by FabianClemenz** [comment]
@jamesgilmorelyst this is resolved in version [3.0.1](https://github.com/farridav/django-jazzmin/releases/tag/v3.0.1)
